chapters/en/slides/module6/module6_01.Rmd

```diff
@@ -0,0 +1,340 @@
+that is driven by commodities,
+such as the need for new pastures for cattle
+or fields to grow crops,
+but this accounts for most of the tropical deforestation.
+
+It would be easy to jump to conclusions
+upon seeing a chart like this 
+and blaming the countries with high local deforestation rate,
+but it that really fair?
+
+How much of this deforestation is really driven by local needs
+versus how much is due to exports to other countries?
+
+---
+
+## A large part of deforestation is driven by exported commodities
+
+```{python}
+# TODO save wrangled data
+deforestation_exported = pd.read_csv('data/share-deforestation-exported.csv', parse_dates=['Year']).query('Entity in ["Brazil", "Indonesia"]') 
+alt.Chart(deforestation_exported).mark_line().encode(
+    y='share_deforestation_exported',
+    x=alt.Y('Year', title=''),
+    color=alt.Color('Entity'))
+```
+
+Notes:
+This chart shows the share of deforestation that is due to exported goods
+from 2005 to 2013.
+
+As we can see,
+a large part of the deforestation in both countries
+is driven by other countries.
+Particularly in Indonesia,
+where around half the country's deforestation
+stems from interest outside Indonesia.
+
+To recap,
+this story has shown us 
+global forest areas are at an all time low
+and still shrinking.
+And although much of the deforestation localized to just a few countries,
+it is driven by global interests.
+
+This could be the end of our story
+if the main aspect we wanted to highlight
+was that much of deforestation is a global problem
+that needs to be solved through international collaborations.
+
+If we wanted to tell a longer story,
+we next as "Which are the commodities that drive most of the deforestation?",
+which is something that the reader might already have in mind since the previous slide.
+Let's explore this next.
+
+---
+
+## Demand for beef is the biggest reason for deforestation in Brazil
+
+```{python}
+# TODO save wrangled data
+brazil_drivers = pd.read_csv('data/drivers-forest-loss-brazil-amazon.csv').query('Year > 2003')
+brazil_drivers = brazil_drivers.melt(id_vars=['Entity', 'Code', 'Year'], var_name='type', value_name='deforestation_hectares').groupby('type').sum().reset_index()
+alt.Chart(brazil_drivers).mark_bar().encode(
+    x='deforestation_hectares',
+    y=alt.Y('type', title='', sort='x'))
+```
+
+Notes:
+This slide shows the biggest drivers of deforestation in Brazil
+between the years 2003 and 2013.
+
+We can see that loss of forest 
+is dominated by the demand for beef,
+which leads to permanent clearing of forest
+to create pastures for cattle grazing.
+
+There is also a smaller contributions from small scale clearings,
+fires, and commercial crops.
+
+Although beef is the major culprit in rain forest deforestation,
+it could be the case that our demand for more food has simply increased
+and cattle herds are an effective use of land to create food for human consumption.
+Let's flip to the next slide to find out!
```

Comment:
I don't think we say anything that is suggesting the it is global beef demand that drives this. I avoided mentioning global and local because the commodity breakdown does not have export and import numbers, those are only available for the total. I can add an explicit sentence about what we are not saying.

Regarding the pasture use, it is [specifically for beef as per the end of this section](https://ourworldindata.org/drivers-of-deforestation#is-soy-production-driving-deforestation). I have not read those references because I think it is outside what we have time for and that Our world in data is generally correct based on what I have fact checked in the past. I can change the entry to say "pasture for cattle" or something like that.

Reply:
Regarding drivers that is what they are called in the linked article and its references as far as I can see, so I don't think we should call them something else. I agree it would be interesting to read the scientific publication and understand how they determine what is a driver and what is not, but not having time for that I would go with how they are classified by those researchers.

Regarding small scale clearings, I looked it up in the publication and it is used for a variety of things:

> Small-scale clearing was identified by its size and postclearing land use (combination of cropland, pasture, orchards, and residences) for older clearings and by size only for the fresh clearings. Median area of loss patches identified as small-scale clearing is 5 ha. Only 24% of small-scale clearing sample pixels fall within the most recent INCRA (National Institute of Colonization and Agrarian Reform) settlement map, which indicates that these small-scale clearings are created not only by smallholders (rural settlers) but also by agroindustrial enterprises.
>
> https://advances.sciencemag.org/content/3/4/e1601047

I'll see if I can add that but I also don't want to get too technical